CondTools/SiStrip/plugins/SiStripBadChannelPatcher.cc

```diff
@@ -0,0 +1,228 @@
+// system include files
+#include <vector>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+// user include files
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibTracker/Records/interface/SiStripDetCablingRcd.h"
+#include "CalibTracker/SiStripCommon/interface/SiStripDetInfoFileReader.h"
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+#include "CondFormats/DataRecord/interface/SiStripBadStripRcd.h"
+#include "CondFormats/SiStripObjects/interface/SiStripBadStrip.h"
+#include "DataFormats/FEDRawData/interface/FEDNumbering.h"
+#include "DataFormats/SiStripCommon/interface/ConstantsForHardwareSystems.h" /* for STRIPS_PER_APV*/
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
This doesn't seem to be needed... but ok

Reply:
removed.